src/Player.js

```diff
@@ -43,6 +43,7 @@ class Player extends Character {
     // WARNING: values must be JSON.stringify-able
     this.metadata = data.metadata || {};
     this.playerClass = null;
+    this.config = data.config || {};
```

Comment:
Unnecessary, this is basically what `meta` is for. You can just use a `config` key inside of meta, e.g., "config.autoloot" (meta key supports dot notation)